projects/shared/src/lib/view/brief/part-of/part-of.component.html

```diff
@@ -0,0 +1,79 @@
+ <!--
+  RERO ILS UI
+   Copyright (C) 2021 RERO
+  
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3 of the License.
+  
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+  
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Remove the extra spaces at the begining of theses lines. It's some weird chars that were copied into a license header sometimes ago, and that are not displayed on Mac OS.

![ksnip_20210204-074333](https://user-images.githubusercontent.com/1557213/106855112-55872a00-66b4-11eb-8f99-1584c40a6b56.png)
.

Reply:
done